downstream/modules/dev-guide/proc-create-role.adoc

```diff
@@ -5,37 +5,49 @@
 = Creating roles
 
 [role="_abstract"]
-Using the `ansible-galaxy` command line tool that comes bundled with Ansible, you can create a role with the `init` command.
+You can create roles using the Ansible Galaxy CLI tool. Role-specific commands can be accessed from the `roles` subcommand.
 
-As an example, the following command will create a role directory structure called `role_name` in the current working directory:
+Standalone roles outside of Collections are still supported, but new roles should be created inside of a Collection to take advantage of all the features {PlatformNameShort} has to offer.
 
+.Procedure
+
+. In your terminal, navigate to the `roles` directory inside a collection.
+. As an example, the following command will create a role directory structure called `role_name` inside the collection created previously:
```

Comment:
Should we give the generic command before showing an example? 
Something like the following:

To create a directory structure for a new role inside the `roles` directory, use the following command:
+
----
$ ansible-galaxy role init <new_role_name>
----

.Example

The following command creates a directory structure called `my_role`.
+
----
$ ansible-galaxy role init my_role
----

The collection now includes the new `my_role` directory inside the `roles` directory:

----
~/.ansible/collections/ansible_collections/<my_namespace>/<my_collection_name>
...
└── roles/
    └── my_role/